apps/qubit/modules/clipboard/actions/saveAction.class.php

```diff
@@ -63,7 +63,7 @@ private function getUniquePassword()
   {
     // Try a max of 100 times before giving up (avoid infinite loops when
     // possible passwords exhausted)
-    for ($i = 0; $i < 100; $i++)
+    for ($i = 0; $i < 100; ++$i)
```

Comment:
I thought this would change the for behavior (adding before starting the count), but it doesn't.